docs/index.rst

```diff
@@ -28,7 +28,7 @@ Hopsworks Documentation
    :maxdepth: 4
    :glob:
 
-   featurestore/featurestore.rst
+   featurestore/featurestore
```

Comment:
what is this change meant to do?

Reply:
I think both are valid syntax. In that file we mix both with extension and without it.